.pre-commit-config.yaml

```diff
@@ -0,0 +1,12 @@
+- repo: https://github.com/pre-commit/mirrors-eslint
+  rev: v8.3.0  # Use the sha / tag you want to point at
+  hooks:
+  - id: eslint
+    files: \.[jt]sx?$  # *.js, *.jsx, *.ts and *.tsx
+    types: [file]
+- repo: git@github.com:Yelp/detect-secrets
+  rev: v1.1.0
+  hooks:
+  - id: detect-secrets
+    args: ['--baseline', '.secrets.baseline']
+    exclude: .*/tests/.*
```

Comment:
```suggestion
    exclude: .*/tests/.*
    
```

Reply:
@wenzowski I'm unsure what you're suggesting here. It appears to be the same.

Reply:
just adding the endline?